### ISSUE TITLE
phpcs tool install squizlabs version but PHPCSStandards is expected

### ISSUE THREAD

**Issue by eclipxe13** [shivammathur/setup-php#809]
**Describe the bug**
When installing the tool `phpcs`, it install old tool from [squizlabs](https://github.com/squizlabs/PHP_CodeSniffer) instead of new tool from [PHPCSStandards](PHPCSStandards/php_codesniffer).
At the [setup-php-action documentation](https://github.com/marketplace/actions/setup-php-action) page, it links `phpcs` to PHPCSStandards.

**Version**
- [ ] I have checked releases, and the bug exists in the latest patch version of `v1` or `v2`.
- [X] `v2`
- [ ] `v1`

**Runners**

- [X] GitHub Hosted
- [ ] Self Hosted

**Operating systems**
Ubuntu  22.04.3  LTS

**PHP versions**
PHP 8.2 & PHP 8.3 (other versions are also affected)

**To Reproduce**
```yml
  phpcs:
    name: Code Style (phpcs)
    runs-on: "ubuntu-latest"
    steps:
      - name: Checkout
        uses: actions/checkout@v3
      - name: Setup PHP
        uses: shivammathur/setup-php@v2
        with:
          php-version: '8.3'
          coverage: none
          tools: phpcs
        env:
          fail-fast: true
      - name: Display phpcs version
        run: phpcs --version
```

**Expected behavior**
To install 
PHP_CodeSniffer version 3.8.0 (stable) by Squiz and PHPCSStandards

**Are you willing to submit a PR?**
I've had check the source code but it looks like the tool already pointing to the right tool (from PHPCSStandards), but the installed version is different. I can try to make a PR but does not know what code to patch.

**Workaround**
Install using phive or composer, like:
```yml
  phpcs:
    name: Code Style (phpcs)
    runs-on: "ubuntu-latest"
    steps:
      - name: Checkout
        uses: actions/checkout@v3
      - name: Setup PHP
        uses: shivammathur/setup-php@v2
        with:
          php-version: '8.3'
          coverage: none
          tools: cs2pr, phive
        env:
          fail-fast: true
      - name: Install phpcs
        run: phive install phpcs --trust-gpg-keys 5E6DDE998AB73B8E
      - name: Code style (phpcs)
        run: tools/phpcs -q --report=checkstyle | cs2pr
```

**Comment by shivammathur** [comment]
It is updated in the develop branch, I will create a release this weekend

**Comment by shivammathur** [comment]
Released [`2.29.0`](https://github.com/shivammathur/setup-php/releases/tag/2.29.0) with the change.

**Comment by LorenzoRogai** [comment]
@shivammathur could you please update the v2 tag to the latest version? GitHub Action is still using "setup-php" from January 15 that doesn't contains the new PHPCSStandards repo
